--- SDL2_image-2.8.3/configure.ac.orig	2024-12-21 18:43:41.349424582 +0100
+++ SDL2_image-2.8.3/configure.ac	2024-12-21 18:47:22.204894772 +0100
@@ -411,10 +411,7 @@ if test x$enable_jpg = xyes; then
                     jpg_lib=[`find_lib "libjpeg*.dll"`]
                     ;;
                 *)
-                    jpg_lib=[`find_lib "libjpeg[0-9]*.so.*"`]
-                    if test x$jpg_lib = x; then
-                        jpg_lib=[`find_lib "libjpeg.so.*"`]
-                    fi
+                    jpg_lib=[`find_lib "libjpeg.so.*"`]
                     ;;
             esac
 
